tests/testcases/Group3-RBAC/3-07-LDAP-usergroup-manage-project-groupmembers.md

```diff
@@ -0,0 +1,58 @@
+Test 3-07 - LDAP usergroup manage project group members
+=======
+# Purpose:
+
+To verify LDAP group can be assigned a role in project member
+
+# References:
+User guide
+
+# Environment:
+
+* This test requires that a Harbor instance is running and available.
+* An LDAP server is running and available, and enabled memberof overlay feature
+* Harbor is set to authenticate against an LDAP or AD server. ( auth_mode is set to **ldap_auth** .) The user data is stored in an LDAP or AD server.
+* LDAP group config parameter are configured.
+    1. ldap_group_basedn
+    1. ldap_group_filter
+    1. ldap_gid 
+    1. ldap_group_scope   
+* A linux host with Docker CLI installed (Docker client).
+
+# Test Steps:
+
+**NOTE:**  
+* Create group  harbor_guest, harbor_dev, harbor_admin in LDAP.
+* Create LDAP user guest_user, dev_user, admin_user in harbor.
+* Assign add group following members
+    1. harbbor_guest --- guest_user, admin_user.
+    1. harbor_dev    --- dev_user.
+    1. harbor_admin  --- admin_user.
+
+* Login as admin user, create project private proj_group_test
+* Add following group with the roles to proj_group_test
+    1. harbor_guest  --- guest, add this member with LDAP Group DN directly: cn=harbor_guest,ou=groups,dc=example,dc=com.
+    2. harbor_dev    --- developer, create user group with LDAP group DN directly: cn=harbor_developer,ou=groups,dc=example,dc=com
+    3. Add a user group: group DN: cn=harbor_admin,ou=groups,dc=example,dc=com, with name harbor_admin, 
+    4. Add project member, select existing user group harbor_admin, assign role administrator.
+* Login user guest_user, dev_user, admin_user in web console. all of them can see the proj_group_test.
+* Login user guest_user, dev_user, admin_user in command line. try to push pull images.
+* Remove admin_user from LDAP group harbor_admin, login again with admin_user. check the role in project proj_group_test
```

Comment:
This step and the following step are manual only right?  can we automate these steps?

Reply:
This testcase can be automated